pmda/util.py

```diff
@@ -177,3 +177,41 @@ def make_balanced_slices(n_frames, n_blocks, start=None, stop=None, step=None):
     slices[-1] = slice(last.start, last_stop, last.step)
 
     return slices
+
+
+def pair_wise_rmsf(mu1, mu2, t1, t2, sos1, sos2):
```

Comment:
Make this function more general: Do not call it `*_rmsf()` but whatever it is called in the paper. This is a general function to combine the means and standard deviations. The test case should also be a general thing where you generate a big random array and then partition the array in different ways. The check is the calculation of mean and variance from the full array.